.github/workflows/posix.yml

```diff
@@ -245,18 +245,21 @@ jobs:
         name: cov-posix-ncp-rcp-migrate
         path: tmp/coverage.info
 
-  posix-pty:
+  pty-linux:
```

Comment:
Can we give the different job instances a more describe name that indicates whether daemon mode is enabled? Currently, the job instances are listed as:

- pty-linux (0)
- pty-linux (1)
- pty-macos (0)
- pty-macos (1)